src/Microsoft.Framework.PackageManager/Commands/InstallCommand.cs

```diff
@@ -23,31 +23,29 @@ public InstallCommand(AddCommand addCmd, RestoreCommand restoreCmd)
             _restoreCommand = restoreCmd;
         }
 
-        public Reports Reports { get; set; }
+        public ILogger Logger { get; set; }
```

Comment:
Yeah, we can do this after we extract `dnu` from `dnx`. Filed an issue: https://github.com/aspnet/dnx/issues/1670. We might need to move this issue to dnx repo later.